spec/features/process_a_request_accept_spec.rb

```diff
@@ -76,6 +76,29 @@
       visit prison_visit_path(vst, locale: 'en')
       expect(page).to have_css('span', text: 'by joe@example.com')
     end
+
+    scenario 'accepting a booking but contact list fails', vcr: {
+      cassette_name: 'process_contact_list_fails',
+      allow_playback_repeats: true
+    } do
+      visit prison_visit_process_path(vst, locale: 'en')
+      click_button 'Process'
+
+      # Renders the form again
+      expect(page).to have_text('Visit details')
```

Comment:
Always prefer have_css with a fairly specialised selector to improve speed and avoid capybara parsing all the text node of the page until it find the actual text